be/src/vec/functions/array/function_array_contains_all.cpp

```diff
@@ -37,8 +37,6 @@ class FunctionArrayContainsAll : public IFunction {
 
     String get_name() const override { return name; }
 
-    bool use_default_implementation_for_nulls() const override { return true; }
```

Comment:
do not modify code like this if this pr just wants to do refactor on `check_set_nullable`

Reply:
> the default value of this function is true

you can make another pr to refactor this instead of doing it in this pr.

Reply:
actually the other reviewer suggest me to remove this code so i doing it here

Reply:
maybe you can just change this pr title to `remove the redundant nullable value calculation in function call` to fit it

Reply:
ok